.github/actions/chocolatey/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+function choco {
+	mono /opt/chocolatey/choco.exe "$@" --allow-unofficial --nocolor
```

Comment:
If you switch to using the official Docker image, usage of `--allow-unofficial` will not be required.